app/main/services/search_service.py

```diff
@@ -157,9 +157,9 @@ def url_for_search(**kwargs):
     except TransportError as e:
         try:
             root_causes = getattr(e, "info", {}).get("error", {}).get("root_cause", {})
```

Comment:
We don't want to overwrite the original exception with `as e`. Attribute errors aren't interesting